docs/backuprestore-with-oadp.md

```diff
@@ -431,7 +437,99 @@ spec:
 > 1. The examples provided are just for reference. Create your own backup and restore CRs based on your needs.
 > 2. The backup and restore CRs must be created in the same namespace where the OADP is installed which is `openshift-adp`.
 > 3. The OADP configmaps should only contain backup and restore CRs which are in YAML format or multiple document YAML format. Other type of CRs are unknown to the LCA and will be ignored.
->
+
+##### 1.1 Backup resource when LVMS is used
+
+When LVMS is used to provide dynamic storage in the cluster, the backup and restore resources for the
+app should also include some extra fields to ensure each PV content is maintained when performing an IBU.
+
+backup_app.yaml
+
+```yaml
+apiVersion: velero.io/v1
+kind: Backup
+metadata:
+  labels:
+    velero.io/storage-location: default
+  name: small-app
+  namespace: openshift-adp
+spec:
+  includedNamespaces:
+  - test
+  includedNamespaceScopedResources:
+  - secrets
+  - persistentvolumeclaims
+  - deployments
+  - statefulsets
+  includedClusterScopedResources:
+  - persistentVolumes              # <- required field
+  - volumesnapshotcontents         # <- required field
+  - logicalvolumes.topolvm.io      # <- required field
+```
+
+restore_app.yaml
+
+```yaml
+apiVersion: velero.io/v1
+kind: Restore
+metadata:
+  name: small-app
+  namespace: openshift-adp
+  labels:
+    velero.io/storage-location: default
+  annotations:
+    lca.openshift.io/apply-wave: "3"
+spec:
+  backupName:
+    small-app
+  restorePVs: true               # <- required field
+  restoreStatus:                 # <- required field
+    includedResources:           # <- required field
+      - logicalvolumes           # <- required field
+```
+
+As a complement, we also included below, sample resources to backup and restore the cluster's manifests required
```

Comment:
Nit: I think it is important to note that this is always required, not restoring LVMCluster will break LVMS, so not sure if its a "complement"

Reply:
totally agree, will move it up in the text and rephrase it to better highlight this, thanks for commenting it